stage1/app-rm/app-rm.go

```diff
@@ -59,7 +69,24 @@ func main() {
 	}
 
 	enterCmd := stage1common.PrepareEnterCmd(false)
+	switch flagPhase {
+	case 1:
+		// phase1: app-rm:phase0 -> app-rm:phase1
+		err = cleanupPhase1(appName, enterCmd)
+	default:
```

Comment:
just a nit, but I suggest to make the set of supported cleanup phases (or cleanup in stages) more explicit and fail for an unsupported phase:

```
switch flagPhase {
  case 0:
     // cleanup things in stage0
  case 1:
    // cleanup things in stage1
  default:
    log.FatalF("unsupported phase %d", flagPhase)
}
```